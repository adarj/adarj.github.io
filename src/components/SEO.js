import React from "react"
import { Helmet } from "react-helmet"

const SEO = () => {
  const title = "Akil Darjean - Software Developer"

  return (
    <Helmet>
      <html lang="en" />
      <title>{title}</title>
    </Helmet>
  )
}

export default SEO
