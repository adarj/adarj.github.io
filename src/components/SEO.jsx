import React from "react"
import { Helmet } from "react-helmet"

const SEO = () => {
  const title = "Akil Darjean"

  return (
    <Helmet>
      <html lang="en" />
      <title>{title}</title>
    </Helmet>
  )
}

export default SEO
