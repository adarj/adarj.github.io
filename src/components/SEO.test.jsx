import React from 'react';
import { shallow } from 'enzyme';

import SEO from './SEO';

describe('SEO', () => {
  it('renders correctly', () => {
    const tree = shallow(<SEO title="John Doe" />);
    expect(tree).toMatchSnapshot();
  });
});

