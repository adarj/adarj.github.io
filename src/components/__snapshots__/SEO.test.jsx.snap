// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SEO renders correctly 1`] = `
<HelmetWrapper
  defer={true}
  encodeSpecialCharacters={true}
>
  <html
    lang="en"
  />
  <title>
    John Doe
  </title>
</HelmetWrapper>
`;
